scripts/jenkins-yoctobuild-build.sh

```diff
@@ -553,6 +558,16 @@ fi
 # Done with server build.
 # -----------------------
 
+# -----------------------
+# Get mender-binary-delta
+# -----------------------
+
+if [ -d $WORKSPACE/meta-mender/meta-mender-commercial ]; then
+    RECIPE=$(ls $WORKSPACE/meta-mender/meta-mender-commercial/recipes-mender/mender-binary-delta | sort | head -n1)
+    mkdir -p $WORKSPACE/mender-binary-delta
+    s3cmd get --recursive s3://$(sed -e 's,delta_,delta/,; s/\.bb$//' <<<$RECIPE)/ $WORKSPACE/mender-binary-delta/
```

Comment:
As discussed privately, we'll need to add the AWS_KEY_... variables to the GitLab env.

Optionally, use `awscli` instead of `s3cmd`

Reply:
Variables added!